pkg/userdata/centos/testdata/docker-1.13-kubelet-v1.11-aws.golden

```diff
@@ -35,14 +33,18 @@ write_files:
     #     mls - Multi Level Security protection.
     SELINUXTYPE=targeted
 
-
 - path: "/etc/sysconfig/kubelet-overwrite"
   content: |
-    KUBELET_EXTRA_ARGS=--cloud-provider=aws --cloud-config=/etc/kubernetes/cloud-config \
-      --authentication-token-webhook=true --hostname-override=node1 --read-only-port 0 \
-      --cluster-dns= --cluster-domain=cluster.local
-
-
+    KUBELET_EXTRA_ARGS=--authentication-token-webhook=true \
+      --cloud-provider=aws \
+      --cloud-config=/etc/kubernetes/cloud-config \
+      --hostname-override=node1 \
+      --read-only-port=0 \
+      --keep-terminated-pod-volumes=false \
```

Comment:
Got marked as deprecated https://github.com/kubernetes/kubernetes/pull/47539

Reply:
Also it seems its default by default

Reply:
yep, to avoid problems like described here: https://github.com/kubernetes/kubernetes/issues/45191 it set to `false`

Reply:
Ah sorry, its `false` by default. So we actually don't even need to set this flag

Reply:
yeah, but CIS requires it to be like that :)